### DISCUSSION TITLE
Champion list

### DISCUSSION THREAD

**Discussion by okty2k** [FarbstoffRSL/RSL-Helper#377] [Q&A]
Would it be possible (maybe while RSL Helper is running) to get the complete list of champions (maybe with level as well)?

**Comment by RedMan1911** [comment]
> Would it be possible (maybe while RSL Helper is running) to get the complete list of champions (maybe with level as well)?

It already exists in RSL Helper if you click on Champion Index. Or do you think something else?

![image](https://user-images.githubusercontent.com/105769051/169159198-41582abd-0814-4ac2-acc1-42cc92ea584e.png)

**Reply by okty2k** [reply]
I know that exists. But I am interested in exporting the data (CSV, xls).  I want to be able to filter by HP, Def, etc. And use multiple such filters
